source/recipecard/cardCarousel.js

```diff
@@ -1,51 +1,137 @@
-// CardCarousel.js
 
 class CardCarousel extends HTMLElement {
     constructor() {
         super();
         this.attachShadow({ mode: 'open' });
+
+        const style = document.createElement('style');
+        style.innerHTML = `
+        .back, .forward {
+            cursor: pointer;
+            position: relative;
+            top: 50%;
+            display: block;
+            width: auto;
+            height: 20px;
+            padding: 17px;
+            border-radius: 50px;
+            color: black;
+            font-size: 20px;
+            overflow: hidden;
+            transition: all .35s;
+          }
+          
+          .back {
+          }
+          
+          /* place forward button to right of page */
+          .forward {
+          }
+          
+          /* button transition features */
+          .back:hover, .forward:hover {
+            color: #df342f;
+          }
+          
+          .back:before, .back:after,
+          .forward:before, .forward:after {  
+            position: absolute;
+            content: "";
+            width: 100%;
+            height: 100%;
+            top: -100%;
+            left: 0;
+            background: lightgrey;
+            z-index: -1;
+            transition: all .35s;
+          }
+          
+          .back:before, .forward:before {
+            opacity: .5;
+          }
+          
+          .back:after, .forward:after {
+            transition-delay: .25s;
+          }
+          
+          .back:hover:before, .back:hover:after,
+          .forward:hover:before, .forward:hover:after {
+            top: 0;
+          }
+          `;
+
+        this.shadowRoot.append(style);
     }
 
 
-    /*
-    * This function will be called in an event listener when the right button is clicked.
-    * The next 3 recipe cards will be shown.
-    */
-    nextCards() {
-        const cards = this.shadowRoot.querySelectorAll('recipe-card');
-        for (let i = cards.length - 4; i >= 0; i--) {
-            if (cards[i].getAttribute('style') != 'display: none') {
-                cards[i].setAttribute('style', 'display: none');
-                cards[i + 3].style.display = '';
-            }
+    // paramter is in an array of recipe cards
+    createCardCarousel(recipeCards) {
+        const forwardButton = document.createElement('a');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group11/cse110-sp21-group11/pullRequest?prid=8549129)
```suggestion
        const forwardButton = document.createElement("a");
```